code/modules/jobs/job_types/head_of_personnel.dm

```diff
@@ -51,3 +51,9 @@
 		/obj/item/melee/classic_baton/telescopic=1, /obj/item/modular_computer/tablet/preset/advanced = 1)
 
 	chameleon_extras = list(/obj/item/gun/energy/e_gun, /obj/item/stamp/hop)
+
+/datum/outfit/job/hop/pre_equip(mob/living/carbon/human/H)
+	..()
+	if(locate(/datum/holiday/ianbirthday) in SSevents.holidays)
+		H.undershirt = /datum/sprite_accessory/undershirt/ian
+		H.update_body()
```

Comment:
you should not need to do this in the pre_equip

Reply:
honestly I'd prefer that you add undershirt to the available types in the datum and then auto call this during equip if it's necessary to show the undershirt on the body

Reply:
sure thing

Reply:
added!